### PR TITLE
User can turn their car 

### PR DESCRIPTION
**Pull request by eniolajohnson**
image

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    car.className = 'north';
+  } else if (event.code === "ArrowDown") {
+    car.className = 'south';
+  } else if (event.code === "ArrowLeft") {
+    car.className = 'west';
+  } else if (event.code === "ArrowRight") {
+    car.className = "east";
+  } else {
+    return;
+  }
```
Comment (uzair-ashraf) [inline main.js:21]: not needed

File: main.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+var car = document.querySelector('img');
+
+var carDirectionData = {
+  north: 'north',
+  south: 'south',
+  west: 'west',
+  east: 'east'
+}
```
Comment (uzair-ashraf) [inline main.js:8]: this kind of doesnt' really work as a data model
You could have a direction property which stores the direction?
`direction: 'east'`

**Review by uzair-ashraf — CHANGES_REQUESTED** [review]

File: main.js (hunk 1 above)
Comment (eniolajohnson) [reply main.js:21]: Alright, will edit.

**Comment by eniolajohnson** [comment]
image

**Comment by eniolajohnson** [comment]
Fixed it.

**Comment by eniolajohnson** [comment]
image

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**